dev_scripts/template_pyxis.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+from __future__ import print_function
+import os
+import sys
+from pprint import pprint
+
+PYXIS_SERVER_URL = "https://pyxis.stage.engineering.redhat.com/"
+
+# Allow imports from parent directory.
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+
+os.environ["FRESHMAKER_DEVELOPER_ENV"] = "1"
+os.environ["FRESHMAKER_CONFIG_FILE"] = os.path.join(sys.path[0], "config.py")
+
+from freshmaker import conf, app
+from freshmaker.pyxis import Pyxis
+
+app_context = app.app_context()
+app_context.__enter__()
+
+if len(sys.argv) != 1:
+    print("Template for testing freshmaker.Pyxis class")
+    print("Usage: ./pyxis.py")
```

Comment:
Perhaps it would be good to add some comment to remember to set `dns_canonicalize_hostname = false`

Reply:
Yes, you are right